src/runtime/analytics-service.js

```diff
@@ -265,14 +266,21 @@ export class AnalyticsService {
     });
   }
 
+  /**
+   * @return {boolean}
+   */
+  shouldLogPublisherEvents_() {
```

Comment:
I believe this makes it so publishers can't log to propensity without specifying logFromPublisherConfig_. Is that to be expected?

Reply:
This makes it so the analytics service only receives events from publisher clients if a configuration setting and an experiment are both active.

This does not effect what propensity gets, that's controlled by propensity-server

Reply:
I see now! LGTM